docs/reference/analysis/overview.asciidoc

```diff
@@ -0,0 +1,75 @@
+
+== Text analysis overview
+++++
+<titleabbrev>Overview</titleabbrev>
+++++
+
+Text analysis enables {es} to perform full-text search, where the search returns
+all _relevant_ results rather than just exact matches.
+
+For example, a full-text search for `Quick fox jumps` should not only return
+documents containing that exact search string but also documents containing
+similar or related words, like  `fast fox` or even `foxes leap`.
+
+[discrete]
+[[tokenization]]
+=== Tokenization
+
+Analysis makes full-text search possible by breaking an text down into smaller
```

Comment:
I'd put the focus on tokenization over analysis, and provide a bit more of the "why". Something like:

Indexing the contents of a field as individual tokens rather than a single string makes full-text search possible. If you index the phrase _the quick brown fox jumps_  as a single string and the user searches for _quick fox_, it isn't considered a match. However, if you tokenize the phrase and index each word separately, the terms in the query string can be looked up individually.

Reply:
Reworded with b47688727e9. I did keep an updated version of the original intro para.